test_regress/t/t_force_chained_bad.out

```diff
@@ -0,0 +1,3 @@
+%Error: t/t_force_chained_bad.v:30:  got='h0 exp='h00000001
+%Error: t/t_force_chained_bad.v:37:  got='h0 exp='h00000002
+%Error: t/t_force_chained_bad.v:43:  got='h0 exp='h00000003
```

Comment:
For the tests that are IEEE legal but Verilator gets wrong, please rename to remove the _bad.  We use _bad to mean something that's illegal code, versus something that only currently fails due to limitations.   Sometimes we use _unsup for that (unsupported), I could live with that but probably would skip in this case since we don't give an unsupported warning.

Reply:
I think adding such unsupported warnings would be valuable. Should I add new V3ErrorCode for such cases to make it suppressible?

Reply:
Assuming no false-positives, that is the warning shows cases where there only are real problems, I would suggest just reuse UNSUPPORTED.

Reply:
Alright. Right now, I'd refrain from adding unsupported checks as I don't want to introduce false positives which are costly to detect.
I'd removed the `bad` suffix from test names.